3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,147 @@
+//////Item array removal//////
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+names.splice(1,1);
```

Comment:
This will only remove the element with the index 1 in the array.  You should create a function that can remove any name that is present on the array.